deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -52,6 +52,8 @@ monitoring:
   # Monitoring requires Prometheus to be pre-installed
   enabled: false
   # -- Whether to create the Prometheus rules for Ceph alerts
+  MetricsDisabled: true
```

helm settings in yaml are `camelCase`
```suggestion
  metricsDisabled: true
```

---

But do we actually want to change this default for all helm users?

---

I see https://github.com/rook/rook/blob/master/Documentation/CRDs/Cluster/ceph-cluster-crd.md  it's default value is false. 

metricsDisabled: Whether to disable the metrics reported by Ceph. If false, the prometheus mgr module and Ceph exporter are enabled. If true, the prometheus mgr module and Ceph exporter are both disabled. Default is false.